### ISSUE TITLE
Object reference error when reordering DataMembers

### ISSUE THREAD

**Issue by oruchreis** [d1820/proto-attributor#4]
Hi,
Thanks for the extension, but I couldn't use with reordering because of "Object reference not set to an instance of object" error. Here is a sample that I want to reorder:
```
    [DataContract(Namespace = Constants.Namespace)]
    public abstract class BaseResponse
    {
        /// <summary>
        /// A summary
        /// </summary>
        [DataMember]
        public string TraceId { get; set; }

        /// <summary>
        /// 
        /// </summary>
        [DataMember]
        public ResponseDiagnostics Diagnostics { get; set; }

        /// <summary>
        /// 
        /// </summary>
        [IgnoreDataMember]
        public bool IsHandled { get; set; }

        /// <summary>
        /// 
        /// </summary>
        [DataMember]
        public List<ResponseMessage> Messages { get; set; }
    }
```

Visual Studio Version 16.8.5

**Comment by d1820** [comment]
@oruchreis  This is happening because that existing example is an invalid protobuf contract. Protobuf requires the order property To apply correctly either remove all attributes and reapply or add the order property so the extension knows what to key off of while generating the correct order numbers. I can work on fail-safeing when trying to use with an invalid contract, but it would not be a high priority at this time.

**Comment by d1820** [comment]
Implemented #5
